gradle/libs.versions.toml

```diff
@@ -78,8 +77,7 @@ paparazzi-gradlePlugin = { module = "app.cash.paparazzi:paparazzi-gradle-plugin"
 jimfs = "com.google.jimfs:jimfs:1.3.0"
 coil-compose = { module = "io.coil-kt.coil3:coil-compose", version.ref = "coil" }
 coil-core = { module = "io.coil-kt.coil3:coil", version.ref = "coil" }
-coil-network = { module = "io.coil-kt.coil3:coil-network", version.ref = "coil" }
-ktor-engine-okhttp = { module = "io.ktor:ktor-client-okhttp", version.ref = "ktor" }
```

Comment:
Abstracted the [network interface](https://github.com/coil-kt/coil/blob/main/coil-network-core/src/commonMain/kotlin/coil3/network/NetworkClient.kt) so we can remove the Ktor dependency or replace the interface entirely in the future.